lib/pages/forms_pages/camera_page.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+import 'package:mosquito_alert_app/pages/forms_pages/adult_report_page.dart';
+import 'package:whatsapp_camera/whatsapp_camera.dart';
+
+class CameraPage extends StatefulWidget {
+
+  CameraPage();
+
+  @override
+  _CameraPageState createState() => _CameraPageState();
+}
+
+class _CameraPageState extends State<CameraPage> {
+  void _openCamera() async {
+    final photos = await Navigator.push(
+      context,
+      MaterialPageRoute(
+        builder: (context) => WhatsappCamera(
+          multiple: true,
+        ),
+      ),
+    );
+
+    if (photos != null && photos.length >= 1 && photos.length <= 3) {
+      await Navigator.push(
+        context,
+        MaterialPageRoute(
+          builder: (context) => AdultReportPage(photos: photos),
+        ),
+      );
+    }
+    Navigator.pop(context);
+  }
```

Comment:
Going to the previous page in case of selecting more than three images without any explanation can be confusing for users. To address this, consider the following solutions:

a) Limit the number of selectable images in the gallery FilePicker to a maximum of three.
b) Configure WhatsAppCamera with `multiple: false`, so users can only add images one by one.
c) Display a dialog when more than three images are selected, prompting the user to unselect the excess images.

Reply:
Let's go with "multiple: false", seems the most straightforward solution. If they need to add more photos they can do it in the next screen